buildenv/ansible/roles/openj9-build-environment/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2018, 2018 Pavel Samolysov
+# Copyright (c) 2018, 2020 Pavel Samolysov
```

Comment:
This isn't an IBM copyright, seems an IBM copyright needs to be added for these changes.

Reply:
Fixed.